src/components/app.test.js

```diff
@@ -1,26 +1,34 @@
-import React from 'react';
-import {shallow, mount} from 'enzyme';
-import App from './app';
-import BasicTutorial from './tutorials/basic-tutorial';
-import DeveloperPreview from './developer-preview';
+import { render } from 'react-dom'
+import React from 'react'
+import { MemoryRouter } from 'react-router-dom'
+import App from './app'
 
-it('renders without crashing', () => {
-  shallow(<App url={document.URL}/>);
-});
```

Comment:
i thought this test was helpful - i could see if App itself could render (and avoid trying to render nested components)